tst/com/amazon/corretto/crypto/provider/test/EvpSignatureSpecificTest.java

```diff
@@ -546,6 +546,7 @@ private static void assertPssParamsEqual(PSSParameterSpec s1, PSSParameterSpec s
 
   @Test
   void testRsaPSSInitSmallKeyAfterSetParameterLargeSaltThrows() throws Exception {
+    if (TestUtil.isFips()) return;
```

Comment:
are we able to adapt this test case to work under FIPS? perhaps by setting `minimallySecureKeyLen = TestUtil.isFips()? 2048 : 4096` and `smallKeySize = minimallySecureKeyLen / 2`?

if not, instead of returning can we skip it with `Assumptions.assumeTrue(!TestUtil.isFips())` and add a comment explaining why we skip this test in FIPS?

Reply:
`Assumptions.assumeTrue` is a much cleaner way of doing it. I'll update the PR.